_data/simple-icons.json

```diff
@@ -8261,7 +8266,7 @@
         {
             "title": "ŠKODA",
             "hex": "4BA82E",
-            "source": "https://en.wikipedia.org/wiki/File:Skoda_Auto_logo_(2011).svg"
+            "source": "https://commons.wikimedia.org/wiki/File:Skoda_-1200x1200.jpg"
```

Comment:
I'm not sure about this change. The first link was, at least, a vector version.

Reply:
I'm going to include the EPS file while you make a decision.

Reply:
As you mentioned, seeklogo is far from being an official source... We might use this one maybe:
https://commons.wikimedia.org/wiki/File:%C5%A0koda_Works_logo.svg

Reply:
sure, that's much better.